tests/2d_cylindrical_TALA_DG/2d_cylindrical_TALA_DG.py

```diff
@@ -205,16 +201,16 @@ def gaussian(center, c, a):
 # passing in the approximation, nullspace and near-nullspace information configured above.
 
 energy_solver = EnergySolver(T, u, approximation, delta_t, ImplicitMidpoint, bcs=temp_bcs)
-energy_solver.solver_parameters['ksp_rtol'] = 1e-4
 
 stokes_solver = StokesSolver(z, T, approximation, bcs=stokes_bcs,
                              nullspace=Z_nullspace, transpose_nullspace=Z_nullspace,
                              near_nullspace=Z_near_nullspace)
 stokes_solver.solver_parameters['snes_rtol'] = 1e-2
 stokes_solver.solver_parameters['fieldsplit_0']['ksp_converged_reason'] = None
-stokes_solver.solver_parameters['fieldsplit_0']['ksp_rtol'] = 1e-3
+stokes_solver.solver_parameters['fieldsplit_0']['ksp_rtol'] = 1e-7
 stokes_solver.solver_parameters['fieldsplit_1']['ksp_converged_reason'] = None
-stokes_solver.solver_parameters['fieldsplit_1']['ksp_rtol'] = 1e-2
+stokes_solver.solver_parameters['fieldsplit_1']['ksp_rtol'] = 1e-5
+
```

Comment:
I ran the case locally on 4 processes with your changes but I deleted these lines here (to use the respective G-ADOPT default values). Execution completed in under an hour and a half, and the `u_rms` diagnostic I obtained yields a relative difference on the order of `1e-6` compared with the CI run here.